module/applications/actor/character-sheet-2.mjs

```diff
@@ -478,6 +495,25 @@ export default class ActorSheet5eCharacter2 extends ActorSheetV2Mixin(ActorSheet
 
   /* -------------------------------------------- */
 
+  /**
+   * Handle deleting an occupant from a facility.
+   * @param {PointerEvent} event  The triggering event.
+   * @returns {Promise<Item5e>|void}
+   * @protected
+   */
+  _onDeleteOccupant(event) {
+    event.stopPropagation();
+    const { facilityId } = event.target.closest("[data-facility-id]")?.dataset ?? {};
+    const { prop } = event.target.closest("[data-prop]")?.dataset ?? {};
+    const { index } = event.target.closest("[data-index]")?.dataset ?? {};
+    const facility = this.actor.items.get(facilityId);
+    if ( !facility || !prop || (index === undefined) ) return;
+    const value = foundry.utils.getProperty(facility, prop).filter((_, i) => i != index);
```

Comment:
```suggestion
    const value = foundry.utils.getProperty(facility, prop).filter((_, i) => i !== index);
```

Reply:
We're comparing a string and number here so using strict equals doesn't work

Reply:
Linter doesn't like it :(. Could always to `i !== Number(index)`